File: 프로그래머스/0/181834. l로 만들기/l로 만들기.js
function solution(myString) {
    const alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k']
    
    for (let char of alphabet) {
        myString = myString.replaceAll(char, 'l');
    }
    
    return myString;
}

File: 프로그래머스/0/181834. l로 만들기/l로 만들기.py
def solution(myString):
    answer = ''
    alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k']
    for i in alphabet:
        myString = myString.replace(i, 'l')
        answer = myString
    return answer